src/com/owncloud/android/ui/adapter/LocalFileListAdapter.java

```diff
@@ -157,7 +166,9 @@ public View getView(int position, View convertView, ViewGroup parent) {
                 		        );
                             fileIcon.setImageDrawable(asyncDrawable);
                             task.execute(file);
-                        }
+                            Log_OC.v(TAG, "Executing task to generate a new thumbnail");
```

Comment:
general question regarding the LOG statements. As for general appliance how about moving the LOG statements into the Task implementation itself so any use of the task would have these statements?